service/resource/configmap/resource_test.go

```diff
@@ -674,20 +673,19 @@ func Test_Resource_CloudConfig_GetDeleteState(t *testing.T) {
 
 func Test_Resource_CloudConfig_GetUpdateState(t *testing.T) {
 	testCases := []struct {
-		Obj                        interface{}
-		CurrentState               interface{}
-		DesiredState               interface{}
-		SetCtxUpdateAllowed        bool
-		SetCtxUpdateNecessary      bool
-		ExpectedConfigMapsToCreate []*apiv1.ConfigMap
-		ExpectedConfigMapsToDelete []*apiv1.ConfigMap
-		ExpectedConfigMapsToUpdate []*apiv1.ConfigMap
-		ExpectedCtxUpdateAllowed   bool
-		ExpectedCtxUpdateNecessary bool
+		Ctx                                  context.Context
+		Obj                                  interface{}
+		CurrentState                         interface{}
+		DesiredState                         interface{}
+		ExpectedConfigMapsToCreate           []*apiv1.ConfigMap
+		ExpectedConfigMapsToDelete           []*apiv1.ConfigMap
+		ExpectedConfigMapsToUpdate           []*apiv1.ConfigMap
+		ExpectedMessageContextConfigMapNames []string
```

Comment:
The tests are just aligned with the update state computation not being blocked anymore.